pkg/spoke/spokeagent.go

```diff
@@ -260,36 +261,38 @@ func (o *SpokeAgentOptions) RunSpokeAgent(ctx context.Context, controllerContext
 	if err != nil {
 		return err
 	}
-	spokeClusterInformers := clusterv1informers.NewSharedInformerFactory(spokeClusterClient, 10*time.Minute)
```

Comment:
I would keep the informers here and move your usage of them into the feature gate block.

Reply:
updated